lib/decompress/zstd_decompress_block.c

```diff
@@ -1277,43 +1534,112 @@ ZSTD_decompressSequencesLong_body(
         }
         RETURN_ERROR_IF(seqNb<seqAdvance, corruption_detected, "");
 
-        /* decode and decompress */
-        for ( ; (BIT_reloadDStream(&(seqState.DStream)) <= BIT_DStream_completed) && (seqNb<nbSeq) ; seqNb++) {
-            seq_t const sequence = ZSTD_decodeSequence(&seqState, isLongOffset);
-            size_t const oneSeqSize = ZSTD_execSequence(op, oend, sequences[(seqNb-ADVANCED_SEQS) & STORED_SEQS_MASK], &litPtr, litEnd, prefixStart, dictStart, dictEnd);
+        /* decompress without stomping litBuffer */
```

Comment:
This variant is used for "large window" files, and for cold-cache dictionaries.
So, in a "regular" benchmark setup, it may not show up.
Which means, the potential impact to performance can be missed.

Ensure benchmarks cover this used case.
(typically, benchmark decompression speed on `enwik8` compressed with `--ultra -22`).